### ISSUE TITLE
Security Level doesn't change from Impossible

### ISSUE THREAD

**Issue by davidmthomsen** [digininja/DVWA#169]
I've been scratching my head forever here. The Security Level on Brute Force won't change from Impossible. I've tried modifying the index.php for brute. Any ideas?

**Comment by digininja** [comment]
Set the level to whatever you want and then watch the cookie to see if it
gets changed to something else on your way to the brute force page.

I know this used to be a problem ages ago but it was fixed, are you running
the latest version?

**Comment by mike-bailey** [comment]
Please note if this doesn't answered in 7 days it'll be closed as non-responsive

**Comment by Varbaek** [comment]
When you update the "settings" file for DVWA, more specifically the security level to e.g. "Low", then the security level will still be "Impossible" if your cookie (named "security" or "security level") is set to that.

Easiest way to fix this, is to either:
a) Set your cookie to "Low", or
b) Change the server-side setting to "Low", then clear your cookies.

**Comment by MedAmr** [comment]
did you find an answer, im getting the same problem!!!

**Comment by mike-bailey** [comment]
There is an answer directly above your comment.

**Comment by eJepStudent** [comment]
I am also seeing this. The cookie's expiration is set to "Session". Any change to the cookie is lost when I close the browser. Changing the config.inc.php file appears to be a workaround. I'm using the current, rolling version of Kali Linux, Apache2 v. 2.4.34 (Debian) and ver. 10.1.29-MariaDB-6+b1 and DVWA ver. 1.10 *Development* (2015-10-08).

**Comment by digininja** [comment]
Try checking what cookies you've got set. I've seen situations where a
cookie gets set for the specific URL so one page can be at one level while
the rest are at another

**Comment by p00j4** [comment]
@Varbaek your answer was a saviour because i could leave knowing "why" 
so I had it working in my normal browser and then hit the direct URL in incognito, it didn't give the results of "low", i checked and reset the security from UI and it looked perfectly set to low but still didn't help, that is where i found that there are two cookie values for key "security" in the local storage where "impossible" was overriding "low" and no matter how many times i set it to low from the dashboard, the impossible had the upper-hand. 
I suspect it to be have copied during direct url hit in incognito. 
long story short. removing the dup entry resolved it
